modules/mpa/mpa.c

```diff
@@ -0,0 +1,194 @@
+/**
+ * @file mpa.c mpa Audio Codec
+ *
+ * Copyright (C) 2016 Symonics GmbH
+ */
+
+#include <re.h>
+#include <baresip.h>
+#include <ctype.h>
+#include <string.h>
+#include "mpa.h"
+#include <mpg123.h>
+
+/**
+ * @defgroup mpa mpa
+ *
+ * The mpa audio codec
+ *
+ * Supported version: 
+ *      libmpg123   1.16.0 or later
+ *      libtwolame  0.3.13 or later
+ *
+ * References:
+ *
+ *    RFC 2250  RTP Payload Format for the mpa Speech and Audio Codec
+ *
+ */
+
+/*
+4.1.17. Registration of MIME media type audio/MPA
+
+   MIME media type name: audio
+
+   MIME subtype name: MPA (MPEG audio)
+
+   Required parameters: None
+
+   Optional parameters:
+        layer: which layer of MPEG audio encoding; permissible values
+        are 1, 2, 3.
+
+        samplerate: the rate at which audio is sampled.  MPEG-1 audio
+        supports sampling rates of 32, 44.1, and 48 kHz; MPEG-2
+        supports sampling rates of 16, 22.05 and 24 kHz.  This parameter
+        is separate from the RTP timestamp clock rate which is always
+        90000 Hz for MPA.
+
+        mode: permissible values are "stereo", "joint_stereo",
+        "single_channel", "dual_channel".  The "channels" parameter
+        does not apply to MPA.  It is undefined to put a number of
+        channels in the SDP rtpmap attribute for MPA.
+
+        bitrate: the data rate for the audio bit stream.
+
+        ptime: RECOMMENDED duration of each packet in milliseconds.
+
+        maxptime: maximum duration of each packet in milliseconds.
+
+        Parameters which are omitted are left to the encoder to choose
+        based on the session bandwidth, configuration information, or
+        other constraints.  The selected layer as well as the sampling
+        rate and mode are indicated in the payload so receivers can
+        process the data without these parameters being specified
+        externally.
+
+   Encoding considerations:
+        This type is only defined for transfer via RTP [RFC 3550].
+
+   Security considerations: See Section 5 of RFC 3555
+
+   Interoperability considerations: none
+
+   Published specification: RFC 3551
+
+   Applications which use this media type:
+        Audio and video streaming and conferencing tools.
+
+*/
+
+
+static struct aucodec mpa = {
+	.pt        = NULL,
```

Comment:
I guess we can use static payload type 14 here?

https://tools.ietf.org/html/rfc3551#section-6